### ISSUE TITLE
New Feature: Leader Boards

### ISSUE THREAD

**Issue by JimTheMan** [JimTheMan/t2#7]
Could have leader board page with leaders for props (or all types of props), views, etc.

All-time, monthly, weekly, daily

**Comment by Faranch** [comment]
I agree with a Leaderboard page, and we could start with a simple Views tracker running on incremental increases based on plays of videos. The leaderboard could be further augmented through the use of our specific Props buttons: one could be on the Board for having the most entertaining video or another can win for having the best comeback and so on. We could even have people winning for the best sumnation of an event IRL a la the Topic Based Rooms, like the election and such.

**Comment by JimTheMan** [comment]
Yeah, I really like this idea. And I want thing that we could just make leaderboards a separate page with subpages that you get to from the header bar (and we could use the font-awesome barchart icon: http://fontawesome.io/icon/bar-chart/
![fa-bar-chart__font_awesome_icons](https://cloud.githubusercontent.com/assets/5354163/20504493/c0f7cfa6-b015-11e6-8eb5-58786741e17d.png)

_BUT_, I think for the sake of building an **mvp** we should do this later and for now focus on the finished off the core cypher room.
